package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "reviews",
+  "version": "1.0.0",
+  "description": "reviews component for FoodiGo app",
+  "main": "server/serverIndex.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "babel": "./node_modules/.bin/webpack -d --watch",
+    "start server": "nodemon server/serverIndex.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FoodiGo/Reviews.git"
+  },
+  "author": "Sara Brandes",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FoodiGo/Reviews/issues"
+  },
+  "homepage": "https://github.com/FoodiGo/Reviews#readme",
+  "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "body-parser": "^1.18.2",
+    "express": "^4.16.2",
+    "install": "^0.10.4",
```

Comment:
What's install? Just want to make sure this wasn't a typo here.

Reply:
Nice catch